ingestion_tools/scripts/importers/key_image.py

```diff
@@ -69,27 +70,53 @@ def get_existing_preview(self) -> tuple[np.ndarray | None, int | None]:
             return data, data.shape[-1]
         return None, None
 
-    def load_annotations(self) -> Generator[np.ndarray, None, None]:
+    def load_point_annotations(self) -> Generator[KeyPointAnnotation, None, None]:
+        vis_config_importer = VisualizationConfigImporter(self.config, {}, parents=self.parents)
         for annotation in AnnotationImporter.finder(self.config, **self.parents):
-            if annotation.shape.lower() not in {"orientedpoint", "point"}:
-                continue
-            annotation_path = annotation.get_output_path()
-            try:
-                annotation_data = annotation.get_output_data(annotation_path)
-                yield annotation_data
-                # We prefer point files over oriented point files, so stop if we just processed that.
-                if annotation.shape.lower() == "point":
-                    break
-            except FileNotFoundError:
-                print(f"Unable to load annotation data for {annotation.get_output_filename(annotation_path)}")
+            is_default = annotation.is_visualization_default
+            if annotation.shape.lower() in {"orientedpoint", "point"} and is_default:
+                annotation_path = annotation.get_output_path()
+                try:
+                    annotation_data = annotation.get_output_data(annotation_path)
+                    color = vis_config_importer.get_annotation_color(annotation, self.get_tomogram())
```

Comment:
Currently, the key photo gets generated before the neuroglancer config gets generated. We should update that ordering [here](https://github.com/chanzuckerberg/cryoet-data-portal-backend/blob/4cfb783d9ca707dc66766ee7a28ac2abddd517e1/ingestion_tools/scripts/importers/utils.py#L58-L59), to satisfy this dependency.